package.json

```diff
@@ -8,32 +8,32 @@
   "author": "Shopify <app-extensions@shopify.com>",
   "license": "MIT",
   "devDependencies": {
-    "@shopify/argo-run": "^0.2.3",
+    "@shopify/argo-run": "^0.3.0-0",
     "@types/prettier": "^2.1.5",
     "@types/yargs": "^15.0.9",
-    "@typescript-eslint/eslint-plugin": "^4.6.0",
-    "@typescript-eslint/parser": "^4.6.0",
-    "eslint": "^7.12.1",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "^7.21.5",
+    "@typescript-eslint/eslint-plugin": "^4.15.0",
+    "@typescript-eslint/parser": "^4.15.0",
+    "eslint": "^7.19.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.3.0",
+    "eslint-plugin-react": "^7.22.0",
     "fs-extra": "^9.0.1",
     "inquirer": "^7.3.3",
     "prettier": "^2.1.2",
     "ts-node": "^9.0.0",
-    "typescript": "^3.9.7",
+    "typescript": "^4.1.0",
     "yargs": "^16.1.0"
   },
   "scripts": {
     "generate": "ts-node ./scripts/generate",
     "build": "argo-run build",
     "server": "yarn start",
     "start": "argo-run serve --port 39351",
-    "lint": "eslint 'src/**'"
+    "lint": "eslint 'src/**'",
+    "type-check": "tsc --pretty"
```

Comment:
Added this for TS projects

Reply:
should we remove this for non ts projects? https://github.com/Shopify/argo-checkout-template/blob/master/scripts/generate/clean-up.ts#L10

Reply:
Yeah I thought about this too, I couldn't figure out an easy way to pass the information of whether this is a TS project through to the clean up the way it's set up right now, and so we already aren't cleaning up a tsconfig.json if it's not a TS project. I feel like this is all kind of throwaway code for the most part so I didn't dig in further :/

Reply:
@lemonmade this is what we're doing on argo-admin-template - we're passing down the template type and then do a clean up of the package.json: https://github.com/Shopify/argo-admin-template/blob/f500f63bd362acc18db7ab1de16323a457a2fc86/scripts/generate/update-package-json.ts#L16

Reply:
@vividviolet this will be a nice improvement once they are merged :D